.gitignore

```diff
@@ -1,6 +1,7 @@
 pkg/
 Gemfile.lock
 Gemfile.local
+vendor/
```

Comment:
This was previously removed in https://github.com/puppetlabs/puppetlabs-dsc/blob/25240bc5b4eae6cd9cc195856d012219053d762b/.gitignore ... and it looks like we'd have to modify the template to enact any change here: https://github.com/puppetlabs/modulesync_configs/blob/windows/moduleroot/.gitignore